uint8[256] gamma
---
bool success